python/cogs/voice.py

```diff
@@ -246,6 +246,46 @@ async def piss(self, ctx: Context):
             after=lambda e: self._disconnect_after_player(ctx, e)
         )
 
+
+    @commands.command()
+    async def ohfuck(self, ctx: Context):
+        """-> Oh Fuc UwU."""
+
+        if not await check_is_playing_invert(ctx):
+            return
+
+        if ctx.voice_client is None:
+            await self.join(ctx)
+
+        fucks = ['./tts/' + item for item in filter(lambda item: 'OF' in item, listdir('tts'))]
+        random.seed()
```

Comment:
no need to seed a random generator when there's only one option

Reply:
Yes I know but what if I wanna add more later.